### PR TITLE
vsphere ipi: add periodic

### PR DESCRIPTION
**Pull request by jcpowermac**

### CODE REVIEW

**Comment by jcpowermac** [comment]
cc:
@abhinavdahiya 
@patrickdillon

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: base
   dev-scripts:
     cluster: https://api.ci.openshift.org
     name: test
     namespace: openshift-kni
     tag: dev-scripts
+  installer:
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml:12]: These will be pulled in by the `tag_specification` -- does something break if you don't explicitly list them here?

Comment (jcpowermac) [reply ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml:12]: @stevekuznetsov I didn't know that worked.  I will update and check

Comment (jcpowermac) [reply ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml]: hmm
```
11m         16m          17      e2e-vsphere-ipi-conf-vsphere-vips.1610c8499e57c315   Pod    spec.containers{test}                    Warning   Failed         kubelet, origin-ci-ig-n-fc9g   Error: ImagePullBackOff
2m          16m          56      e2e-vsphere-ipi-conf-vsphere-vips.1610c8499e576b95   Pod    spec.containers{test}                    Normal    BackOff        kubelet, origin-ci-ig-n-fc9g   Back-off pulling image "docker-registry.default.svc:5000/ci-op-y7mfr5h7/stable@sha256:e984a8e70231118a960def8b999177bfa3060d82cf1486b730bac144a04f0e12"
```

Comment (jcpowermac) [reply ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml]: @stevekuznetsov I am missing something?

15m         16m          3       e2e-vsphere-ipi-conf-vsphere-vips.1610c8462fe266e9   Pod    spec.containers{test}                    Warning   Failed         kubelet, origin-ci-ig-n-fc9g   Failed to pull image "docker-registry.default.svc:5000/ci-op-y7mfr5h7/stable@sha256:e984a8e70231118a960def8b999177bfa3060d82cf1486b730bac144a04f0e12": rpc error: code = Unknown desc = unauthorized: authentication required

Comment (jcpowermac) [reply ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml:12]: @stevekuznetsov I added the images back and at least there are no auth failures.  I don't really expect vSphere IPI to pass all the tests atm

**Comment by jcpowermac** [comment]
@stevekuznetsov any chance we can get this merged then circle back to clean it up?
https://github.com/openshift/release/pull/9332

**Comment by jcpowermac** [comment]
I think this is stuck somehow...

/test all

**Comment by jcpowermac** [comment]
@stevekuznetsov I added only the `upi-installer` image which is not in the release. Can we get this merged?

**Comment by jcpowermac** [comment]
@stevekuznetsov can I get an approval? We really need to get periodics running for vSphere IPI. Looks like 4.5 is failing and had no notifications.

**Comment by jcpowermac** [comment]
/close

https://github.com/openshift/release/pull/11348

**Pull request closed without merging**